k8s-manifests/ci/hce-cluster-scope.yaml

```diff
@@ -808,15 +811,15 @@ spec:
             - name: CONTAINER_RUNTIME_EXECUTOR
               value: "k8sapi"
             - name: HUB_BRANCH_NAME
```

Comment:
Just a thought, for ci manifest we can keep main branch of hub

Reply:
This is for OSS hub, will be removed after. Not using it in HCE Onprem.